official/resnet/resnet_run_loop.py

```diff
@@ -526,7 +526,7 @@ def resnet_main(
       This is only used if flags_obj.export_dir is passed.
 
   Returns:
-    Dict of results of the run.
+    Dict of results of the run. eval_results and train_hooks.
```

Comment:
Clarify what this means. "Dict of results of the run, with keys 'eval_results' and 'train_hooks'". And similarly in imagenet_main.py.